cli/src/main/package/bin/ide

```diff
@@ -15,6 +15,7 @@ ide_env="$("${_IDEASY}" env --bash)"
 if [ $? = 0 ]; then
   eval "${ide_env}"
   if [ $# = 0 ]; then
+    ide status
```

Comment:
nice.
Currently I think that we should then also change this here:
https://github.com/devonfw/IDEasy/blob/d2a37608db9c70740d2e32332bdf3a8690180c0c/cli/src/main/package/setup#L23

How about calling `ide -o` instead of `ide` here.
This should have the following effect:
* when I open a new shell, it starts faster since no network overhead is triggered pulling from git.
* I do not get additional status messages that could be considered as "spam" if the user is currently not interested in messages from IDEasy but just wants to do something in his shell.

Reply:
OK, I will do that as part of #779 so we can merge this PR without further delay.